process.h

```diff
@@ -143,4 +143,13 @@ void refreshconninode ();
 
 void procclean ();
 
+void getkbps (Process * curproc, float * recvd, float * sent);
```

Comment:
It's not entirely obvious to me whether this is a presentation issue or a library issue.. perhaps put them in their own file at least?